### PR TITLE
Adds better reading for Capacity Update logs

### PR DESCRIPTION
**Pull request by Dominik1999**
more to #270

### CODE REVIEW

**Comment by Dominik1999** [comment]
- we could also add a new function in libs `log_message` analogous to `log_event`
- we could also try to find out how to format in `structlog` in general

**Comment by palango** [comment]
>     * we could also try to find out how to format in `structlog` in general

Good point, I think it would be better to handle this in one place than to have conversions everywhere.  This should be possible by writing a [structlog processor](http://www.structlog.org/en/stable/processors.html)

File: src/raiden_libs/logging.py [hunk 1]
```diff
@@ -34,14 +35,22 @@ def setup_logging(log_level: str) -> None:
     )
 
 
-def log_event(event: Event) -> Dict[str, Any]:
-    event_data = asdict(event)
+def change_bytes(event_dict: Dict, key: Any, val: bytes) -> Dict[str, Any]:
```
Comment (palango) [inline src/raiden_libs/logging.py]: Please add a test for this.

File: src/raiden_libs/logging.py [hunk 2]
```diff
@@ -34,14 +35,22 @@ def setup_logging(log_level: str) -> None:
...
 
-    for key, val in event_data.items():
-        if isinstance(val, bytes):
-            if len(val) == 20:
-                event_data[key] = to_checksum_address(val)
-            else:
-                event_data[key] = to_hex(val)
 
-    return event_data
+def format_to_hex(_logger: Any, _log_method: Any, event_dict: Dict) -> Dict[str, Any]:
```
Comment (palango) [inline src/raiden_libs/logging.py]: Please add a test as well

File: src/raiden_libs/logging.py [hunk 3]
```diff
@@ -34,14 +35,22 @@ def setup_logging(log_level: str) -> None:
...
+    for key, val in event_dict.items():
+        if isinstance(val, bytes):
+            change_bytes(event_dict, key, val)
+        if isinstance(val, Event):
```
Comment (palango) [inline src/raiden_libs/logging.py]: In this case it would be nice to output the event name as well

File: src/raiden_libs/logging.py (hunk 1 above)
Comment (Dominik1999) [reply src/raiden_libs/logging.py]: done

File: src/raiden_libs/logging.py (hunk 2 above)
Comment (Dominik1999) [reply src/raiden_libs/logging.py]: done

File: src/raiden_libs/logging.py (hunk 3 above)
Comment (Dominik1999) [reply src/raiden_libs/logging.py]: Wouldn't that be another logging processor? This should only change bytes to hex and neither delete or add any info, right?

Comment (palango) [reply src/raiden_libs/logging.py]: but you're changing it here: `event_data = asdict(val)`

Comment (Dominik1999) [reply src/raiden_libs/logging.py]: So when I understand you correctly, then you suggest to add another structlog processor that also includes the Event name. Or would you inlcude that in the `format_to_hex` function?

Comment (palango) [reply src/raiden_libs/logging.py]: I'd include it there, as you loose the name when you convert it to a dictionary.

File: tests/pathfinding/test_service.py [hunk 4]
```diff
@@ -333,3 +335,29 @@ def test_invalid_fee_update(pathfinding_service_mock, token_network_model):
     # bad/missing signature
     with pytest.raises(exceptions.InvalidFeeUpdate):
         pathfinding_service_mock.on_fee_update(fee_update)
+
+
+def test_logging_processor():
+    # test if our logging processor changes bytes to checksum addresses
+    # even if bytes-addresses are entangeled into dicts
```
Comment (palango) [inline tests/pathfinding/test_service.py]: You never test with a dict.

Comment (Dominik1999) [reply tests/pathfinding/test_service.py]: ok good point!

File: src/raiden_libs/logging.py (hunk 3 above)
Comment (Dominik1999) [reply src/raiden_libs/logging.py]: finally successfully implemented

**Comment by palango** [comment]
@Dominik1999 you need to rebase again!

**Comment by Dominik1999** [comment]
> @Dominik1999 you need to rebase again!

I seem to be able to merge without rebasing. Should I rebase anyway?

**Comment by palango** [comment]
If you can go for it.

**Pull request merged**